zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -61,6 +61,7 @@
   Date dateUpdated;
   private Map<String, Object> config; // paragraph configs like isOpen, colWidth, etc
   public final GUI settings;          // form and parameter settings
+  private Map<String, Set<String>> runtimeInfos;
```

Comment:
I dont want to persist this info in json.
When i add the `transient` modifier, the serailization to JSON(for broadcasting to client browser) doesnot happen.
I  can add an Exclusion strategy for all the storage handlers. But wanted to check if there is an easier way to achieve this . @Leemoonsoo @cloverhearts